src/hooks/useSearch.js

```diff
@@ -5,14 +5,23 @@ export const useSearch = () => {
   const [hotels, setHotels] = useState([])
 
   const search = async ({ city, startDate, endDate, adults, children }) => {
-    const response = await axios.get(global.API_BASE_URL + 'api/hotels', {
-      params: {
+    let data = {}
+    if (!startDate || !endDate) {
+      data = {
+        city,
+      }
+    } else {
+      data = {
         city,
-        startDate: new Date(startDate).toISOString(),
-        endDate: new Date(endDate).toISOString(),
+        startDate,
+        endDate,
         adults,
         children,
-      },
+      }
+    }
+
+    const response = await axios.get(global.API_BASE_URL + 'api/hotels', {
```

Comment:
Here you should add try and catch Block, not above

Reply:
That's right